src/main/java/com/theironyard/war/Game.java

```diff
@@ -0,0 +1,126 @@
+package com.theironyard.war;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Created by Nigel on 6/24/16.
+ */
+public class Game {
+    Player player1;
+    Player player2;
+    public static final int TOPOFDECK = 0;
```

Comment:
This is interesting.  I assume this makes it so you don't have 0's littered through your code.